fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -794,12 +794,19 @@ public PlanFragment visitPhysicalWindow(PhysicalWindow<? extends Plan> physicalW
 
         if (partitionExprs.isEmpty() && orderByElements.isEmpty()) {
             if (inputPlanFragment.isPartitioned()) {
-                inputPlanFragment = createParentFragment(inputPlanFragment, DataPartition.UNPARTITIONED, context);
```

Comment:
when fragment.root is exchange, it is wrong to call createParentFragment(fragment, ...)
How about move this logical into createParentFragment, or remove this function.
it is error prone